Core/StartupArgManager.cs

```diff
@@ -37,5 +39,33 @@ public static string[] MergeStartupArgs(IEnumerable<string> cmdArgs)
                 return startupArgs.ToArray();
             }
         }
+
+        private static void MigrateArgsFile()
+        {
+            const string ObsoleteFile = "laargs.txt";
+
+            try
+            {
+                if (File.Exists(ObsoleteFile) && string.IsNullOrWhiteSpace(File.ReadAllText(ObsoleteFile)))
+                {
+                    File.Delete(ObsoleteFile);
+                    ConsoleUtil.WriteLine("Obsolete configuration file 'laargs.txt' is empty and has been deleted.");
+                    return;
+                }
+
+                if (File.Exists(StartupArgsPath))
+                {
+                    ConsoleUtil.WriteLine("Unable to migrate your old 'laargs' configuration. The destination file already exists.");
+                    return;
+                }
+
+                File.Move(ObsoleteFile, StartupArgsPath);
```

Comment:
It will throw an exception every time LA is launched when none of the files (old and new) exist.

Reply:
Make this a try-catch?

Reply:
I'll use a try-catch and see what you think.

Reply:
ok. Working on a solution...